lib/repl.js

```diff
@@ -100,14 +108,38 @@ ${ text }
 })();`;
 }
 
+function step(withHooks = false, actions = commands) {
+    if (commands[0].includes('openBrowser()')) actions = actions.slice(0, -1);
+    if (commands[commands.length - 1].includes('closeBrowser()')) actions = actions.slice(1);
+    const actionsString = actions.map(e => {
+        if (!e.endsWith(';')) e += ';';
+        return isTaikoFunc(e) ? '\tawait ' + e : '\t' + e;
+    }).join('\n');
+    const hooks = 'beforeSuite(async () => openBrowser())' +
```

Comment:
should be `await openBrowser();` and `await closeBrowser();`

Reply:
Not necessarily. It's been invoked as lambda function.
Ref https://github.com/getgauge/gauge-repository/blob/397ac48c9bdd52b35cea57b75773315b97565989/templates/js/tests/step_implementation.js#L9